sql/moz-fx-data-marketing-prod/ga_derived/mobile_installs_adjust_v1/query.sql

```diff
@@ -0,0 +1,63 @@
+SELECT
+  date,
+  app,
+  network,
+  campaign,
+  adgroup,
+  creative,
+  CASE
+  WHEN
+    country = 'us'
+  THEN
+    'United States'
+  WHEN
+    country = 'de'
+  THEN
+    'Germany'
+  WHEN
+    country = 'it'
+  THEN
+    'Italy'
+  WHEN
+    country = 'uk'
+  THEN
+    'United Kingdom'
+  WHEN
+    country = 'ca'
+  THEN
+    'Canada'
+  WHEN
+    country = 'es'
+  THEN
+    'Spain'
+  WHEN
+    country = 'pl'
+  THEN
+    'Poland'
+  WHEN
+    country = 'fr'
+  THEN
+    'France'
+  ELSE
+    'Other'
+  END
+  AS country,
+  os,
+  device_type,
+  sum(clicks) AS clicks,
+  SUM(installs) AS installs
+FROM
+  `ga-mozilla-org-prod-001.Adjust.deliverable_*`
```

Comment:
I don't think the automation accounts will have permission to read the source tables from `ga-mozilla-org-prod-001.Adjust`, so:
- This query should be added to the [dryrun skip list](https://github.com/mozilla/bigquery-etl/blob/db2f8dacad3e789a0074c623daf914f271321c06/bigquery_etl/dryrun.py#L35).  I'd suggest looking for the `sql/moz-fx-data-marketing-prod/acoustic/**/*.sql` exclusion and adding another one just like it right underneath for `adjust_derived`.
- This query should have a `schema.yaml` file so the table can be deployed, because the automation won't be able to dryrun the query to determine the schema.  You can do this by running `gcloud auth login --update-adc` and then `bqetl query schema update --use-cloud-function=false path/to/query.sql`.

Reply:
@sean-rose is the schema update script supposed to create a physical yaml file? I think it is broken for me. I get the output below when I run it.

`Richards-MacBook-Pro:bigquery-etl richardbaffour-awuah$ ./bqetl query schema update --use-cloud-function=false sql/moz-fx-data-marketing-prod/adjust_derived/firefox_mobile_installs_v1/query.sql
Apr 24, 2023 5:56:11 PM com.google.zetasql.io.grpc.NameResolverRegistry getDefaultRegistry
WARNING: No NameResolverProviders found via ServiceLoader, including for DNS. This is probably due to a broken build. If using ProGuard, check your configuration
telemetry_derived/foo/query.sql                             OK
sql/moz-fx-data-marketing-prod/adjust_derived/firefox_mobile_installs_v1/query.sql dry runs are skipped. Cannot update schemas.`

Reply:
Yes, it should create the `schema.yaml` file.  Try adding `--ignore-dryrun-skip` to that command's arguments.